azure/durable_functions/orchestrator.py

```diff
@@ -146,13 +146,11 @@ def _reset_timestamp(self):
         decision_started_events = [e_ for e_ in self.durable_context.histories
```

Comment:
This method is currently called `_reset_timestamp()`, which seems incorrect. I believe `_update_timestamp()` would be more accurate.